test/components/WidgetSaveButtonTest.js

```diff
@@ -9,12 +9,12 @@ var DashboardActions = require('../../src/actions/DashboardActions');
 describe('WidgetSaveButton', function () {
     it('shows renders a button', function () {
         var widget = TestUtils.renderIntoDocument(<WidgetSaveButton enabled={true} />);
-        var button = TestUtils.findRenderedDOMComponentWithTag(widget, "button");
+        var button = TestUtils.findRenderedDOMComponentWithTag(widget, "a");
```

Comment:
This assertion is in an 'it' with the message of 'shows renders a button'. 1. It english no good. 2. It should probably match the assertion. What is it checking by checking there is an a tag. Maybe the assertion should be more specific?